### ISSUE TITLE
Article XV - safeguarding property rights

### ISSUE THREAD

**Issue by lapapanite** [EOS-Mainnet/governance#13]
Article XV of the constitution propose eventual auction of inactive accounts and distribution of the proceeds to the community.
After some brainstorming on the issue with @Chrisplove we propose to make a distinction between the property rights at stake. Our intended goals are to safeguard property rights but avoid having inactive accounts blocking the access of the community to essential onchain resources.

Propositions :
1 Claims on onchain resources like RAM, bandwidth should be extinguished for inactive accounts.
2 Claims on onchain rights could be handled with limitations of property rights in their terms of use within the defining smart contract (i.e. brand or commercial name can be held only if account stays active)
3 Tokens and onchain rights on off-chain property (i.e. house or car registry dapp) should be safeguarded. Auctioning those accounts is a clear infringement to property right. A change to those  accounts may have implications on the transfer of property rights of assets standing outside of the chain for instance, it may imply the transfer of a house which may be encumbered with mortgage or tax liabilities, or with inheritance rights.

Under this framework, any account inactive for 3 years will go in an dormant mode which triggers the release of all associated onchain staked resources, but all the property right held by the account will remain valid unless stated otherwise by its defining smart contract.

Thus property rights are protected and we prevent inactive accounts from making a dent on critical onchain resources.

**Comment by cryptonut** [comment]
I agree. I took part (ok, partly took over) the debate in telegram chat and this solution is in line with what I was proposing. I agree it is a simple but elegant approach)

**Comment by tkaraivanov** [comment]
I'm very much in favor of this proposal. I just want to reiterate this part:

> all the property right held by the account will remain valid unless stated otherwise by its defining smart contract.

Many of the arguments about keeping the current article XV imply that we need a way to reclaim unused property (e.g. tokenized house, etc.). This can be moved to the smart contract layer, where we have all the flexibility we need for the different types of assets. This proposed solution does just that - great work, IMO.

**Comment by chrisplover** [comment]
Article XV - Termination of Agreement

A Member is automatically released from all revocable obligations under this Constitution 7 years after the last transaction signed by that Member is incorporated into the blockchain. Their account will be marked as Dormant and all EOS tokens will be converted to EOSD  (Dormant) tokens according to the system contract provisions then in effect. EOSD tokens can be converted back to EOS tokens at a rate of 0.9 via a reactivation process, upon which the obligations of the latest Constitution also take effect.

**Comment by samupaha** [comment]
Here is an idea: Could we let users decide themselves how their account will be handled when it has been inactive for certain time?

The default could be similar to current article in the constitution: "If my account has been inactive for three years, the community may take all the tokens it has and sell the account name."

Another option: "This is a long-term hodling account, I'm keeping this account and there might be long times when I don't do any transactions. If this account has been inactive for three years, the community may charge 1 EOS per year to keep it existing in the blockchain. When this account has less than 1 EOS left, the community may take and sell it."

When there is fixed price for preserving the account, it's easy to see when it's empty and it can be sold. This way we can get rid of truly abandoned accounts, which are used for testing etc. These contain only dust amount of tokens, so nothing is really lost even if the real owner comes back later.

**Comment by BenGates1985** [comment]
Hi Guys,

I posted this in the EosGov telegram chat and was asked by someone to repost it here. 

Fan he written a good article but I still believe the current suggestion is rather rigid. Someone correct me if I'm wrong but wouldn't this be a good compromise?

- a person (or their beneficiaries) owns the property indefinitely.

- if no action is taken on the account for 3 years then the resources can be used for the benefit of the community. By this for example the tokens in the wallet could be rented out to dapp developers and the proceeds placed in a community fund. 

I feel this approach rewards, rather than costs the community. Under these circumstances there really appears no need to permanently confiscate the property. 

- if the account holder returns to claim the property in the future then they would be able to after giving a period of notice so as not to prejudice the dapp developer utilising the tokens.

Once identities have been linked to accounts in the future then there should also be scope to locate owners of abandoned accounts.

**Comment by villesundell** [comment]
The renting idea above is indeed interesting!

I was thinking more towards delegating the power ultimately to the ricardian contract of the account.
So, the Constitution would have something like:
"If freeing the funds and assets are not specified in the Ricardian Contract of the Account, funds and assets are freed after 3 years of inactivity".

Personally, I hold some assets for years without using the chain in question, so I think that "infinite hodling" should be possible, if desired.

**Comment by chrisplover** [comment]
Ive had a think and come round now to the view of Dan’s vision (that Aneta supported) on that inactivity clause. My logic : inactive accounts mean less value is created on EOS, which means tokens are in less demand, which means prices fall, which means BPs go out of business, which means the end. We can figure out details later, but won me over. Value creation by good DApps and business adoption is needed to (at least) offset the inflation if the platform is to be successful. That means terminating dead wood somehow. Just need to do it accurately and fairly.

**Comment by chrisplover** [comment]
So we need to think of another simple way,but something must be there.

**Comment by chrisplover** [comment]
The cost of implementing any solution for this issue (if we feel there is something better than the current constitution), needs to somehow be paid for by the inactive accounts that create the need for that solution.

Eg if we have a DApp that sweeps for inactive accounts and leases them via Chintai, then the creators of the DApp need to be rewarded a certain amount, and some of it should go to the EOS community.

**Comment by BenGates1985** [comment]
I pretty much stick to my viewpoint on this. I don't see why we can't just lease the tokens from inactive accounts and the proceeds of that goes into a community fund. Also to keep costs down the eos in inactive accounts could be moved to one "trust account" then if they owner turns up they could regain access to their tokens through arbitration.

I also think this will eventually result in more activity. If someone is a long term hodler and doesn't feel the need to log in then after 3 years their tokens can be leased to dapp developers. I'm still and will always be (unless someone can give me a good argument for the contrary) in favour of indefinite rights to hold your property. 

If the resources can be utilised for the benefit of the community after 3 years inactivity (which they can) then I see no reason to permanently prevent someone from regaining this property at a future date.

**Comment by villesundell** [comment]
I would give my figurative 2 EOS to @BenGates1985's proposal :+1: It would both, guard indefinite rights, and help community.

There are many "normal" jurisdictions where "ownership" (or the possibility to define how the assets are used) is indefinite, for example perpetual trusts are a way for a trustor to dictate "forever" how the assets are used.

**Comment by chrisplover** [comment]
The lease solution is a good basis to discuss eg leased via chintai. However, we need to solve who is going to design and code this solution  and how they are incentivised to do so. No one will build it for free.

**Comment by chrisplover** [comment]
Property is defined by contract. The question is whether the contract is fair and acceptable. So, let’s design a process, that includes the incentive for those that will build it.

**Comment by tkaraivanov** [comment]
We should gather some statistics first before deciding on a design. It's nothing that will require attention in the next few years. Statistics will help us with assessing the need to free up resources, the need for an additional community fund, etc.

**Comment by BenGates1985** [comment]
The incentive could be paid for by a percentage of the lease revenue perhaps?

**Comment by chrisplover** [comment]
Bingo! There could be a few methods though and also needs to be something that can be subscribed to as to avoid having to change core eos code.

**Comment by samupaha** [comment]
The Constitution should be as simple as possible. That's the main reason why I'm against token leasing from inactive accounts. The article in the Constitution and it's consequences should be simple and easy to understand.

Another reason is that by taking the tokens, EOS will itself become a player in the token rental markets. This will drive the price of leasing down. It makes leasing more attractive than it otherwise would be, which is a bad thing in my mind. It encourages renting rather than owning, which means there is less skin in the game.

**Comment by BenGates1985** [comment]
I respectfully disagree for a few reasons.

1) it doesn't solve the indefinite ownership problem if accounts are auctioned.

2) losing the resources of ones account through leasing should be a sufficient disincentive to keep these inactive accounts to a minimum.

3) Revenue from the inactive accounts' leased tokens can go straight back to the community and go to support additional projects which in turn would make EOS more valuable.

I understand the concerns but I personally would like a situation where ownership can be indefinite. Perhaps leasing is not the ideal approach but at the moment I see no other viable alternative which doesn't confiscate accounts.

**Comment by villesundell** [comment]
If the consensus is to have perpetual property rights, and a simple constitution, the power could be delegated to the ricardian contract of the said account? Then the constitution would be simple, and at the same time the account can define perpetual ownership, if needed. The default could be something (like 3 years) if not defined in the account's ricardian contact.

I am still for the lease, though :laughing:

**Comment by asdfzxh8** [comment]
it's not interesting or not

**Comment by quantumas** [comment]
This could even be automated fully with zero reliance on subjective arbitration in same manner how votes decay over time so that abandoned accounts votes don't count in perpetuity but requires a refresh. The only issue are resources others want to use, lower supply of base coin from inactivity is not harmful to the platform as everything is at most measured as % of vested coins, not all coins.

In 3 years w/ 0 actions that doesn't involve losing coins:
1) unvest all coins, releasing network/bandwidth and cpu resources back to community
2) lose access to alias name (can be replaced with pseudorandom generated name or even just public key)
3) sell ram and storage at market price

That's it. It's really not a complicated issue. Many already suspect account and wallet are separate things.
